manifest_staging/charts/csi-secrets-store-provider-azure/templates/provider-azure-installer.yaml

```diff
@@ -15,6 +15,12 @@ spec:
     metadata:
 {{ include "sscdpa.labels" . | indent 6 }}
     spec:
+      {{- if .Values.imagePullSecrets }}
+      imagePullSecrets:
+      {{- range .Values.imagePullSecrets }}
```

Comment:
We can simplify this to `{{ toYaml .Values.imagePullSecrets | indent 8 }}` instead of range?